gooey/gui/util/functional.py

```diff
@@ -1,6 +1,30 @@
+'''
+Utils for functional methodologies throughout Gooey
 
+'''
 
 
+def merge_two_dictionaries(x,y):

```

Comment:
I think just `merge` is ok as a name. The "two" part can be inferred from the number of arguments ^_^

Reply:
Resolved 😄